### DISCUSSION TITLE
v2: content of App.js and App.d.ts are keep changing

### DISCUSSION THREAD

**Discussion by kybin** [wailsapp/wails#1524] [General]
If I don't add/remove my App's methods, their content shouldn't be changed 
but it seems like their method order is keep changed whenever Wails rebuilds the app.
Which is cumbersome especially when with git.

Make them sorted in alphabetical order would be better.

**Comment by leaanthony** [comment]
Good call. Please open a ticket 👍

**Reply by kybin** [reply]
Done. Thanks!
